docs/cep/table/index.md

```diff
@@ -74,12 +74,21 @@ The following parameters are configured in a table definition:
 | GLOBAL or LOCAL      | Whether the table is globally or locally replicated. Default is `GLOBAL`. |
 | attribute name   | The schema of the table is defined by its attributes with uniquely identifiable attribute names (`camelCase` is used for attribute names as a convention.)|    |
 | attribute type   | The type of each attribute defined in the schema.  This can be `STRING`, `INT`, `LONG`, `DOUBLE`, `FLOAT`, `BOOL`, or `OBJECT`.     |
-| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      | _from         | STRING              | Yes      |
-| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. | _to      | STRING              | Yes      |
+| from        | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a source node of the edge.      |
+| to          | If `collection.type` is specified as `edge`, this field indicates which field to be considered as a destination node of the edge. |
 | [WITH (property_name = expression [, ...])] | Optional properties for the new table, such as a time-to-live or a partition key. |
 | SELECT select_expr [, ...] | The selection criteria for the new table. |
 | FROM from_stream … | The name of the existing stream to select data from. |
```

Comment:
@diana-macrometa - This is an explanation of the statements parts but not parameters.

Reply:
I see. What should I call it, then? I think it is helpful to explain all the parts of our definitions.

Reply:
@diana-macrometa - Please review my changes?

Reply:
They look good to me. Is this PR ready to approve and merge?